src/Resources/config/services/event_listener.xml

```diff
@@ -39,5 +39,11 @@
             <tag name="messenger.message_handler" priority="1" />
         </service>
 
+        <service id="BitBag\SyliusMolliePlugin\EventListener\RefundPaymentGeneratedAutoCompleteListener">
```

Comment:
it's better to use one way of giving service IDs, currently you have one as namespace and the rest as key